cmake/install-emscripten.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+version="$1"
+
+# Ensure we have emscripten installed and activated
+pushd .cache
```

Comment:
this little script gives you a local copy of emscripten in the .cache folder. this felt like the best way to ensure it was there without forcing manual configuration.